README.md

```diff
@@ -174,6 +174,15 @@ Currently available `compiler/version` selections are:
 
 * `gnu/9.3.0` and `gnu/11.2.0`
 * `intel/2021.4.0`
+* `nvhpc/22.1` (use with `--with-gpu` on ac's GPU partition)
+
+*Please note: at the moment GPU-builds need to be done on the compute node.*
+
+On ac, an interactive session on one GPU-node can be allocated with the following command:
```

Comment:
Really minor niggle, but this tripped me up when reading docs. Could we maybe capitalise `AC` to indicate we mean a machine/partition, and put something like `Bull Sequana... (AC)` in the header, so that it's clear what we're referring to?

Reply:
That makes sense, happy to make the change

Reply:
Wouldn't limit the section to AC, though, as it's applicable to all four partitions. Only the GPU-specific stuff relates to AC